trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/veldis.py

```diff
@@ -0,0 +1,88 @@
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+# veldis.py 
+#
+# Created:  Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units
+import numpy as np
+
+# ----------------------------------------------------------------------
+# veldis
+# ----------------------------------------------------------------------
+
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+def veldis(qg,x,y,xbar,ybar,st,ct,alpha,Re,npanel):
+    """Compute the tangential velocity distribution at the       
+                 midpoint of each panel   
+
+    Assumptions:
```

Comment:
source?

Reply:
There is no source. Updated the file and docstring though